lib/hooks/resolve/index.js

```diff
@@ -32,11 +33,5 @@ module.exports = settings => {
   return model => {
     return queue(model.data)
       .then(response => pollChangelog(response.MessageId));
```

Comment:
@lennym @joefitter so leave it like this?

Reply:
this comment seems like it's in response to something, although I don't know what that is :)

Reply:
Click the "show outdated" link above to reopen the previous convo.

Reply:
ah yeah, leave it without the `if`

Reply:
soz I didn't know this comment was in reference to removing that

Reply:
no worries :D 

Am I good to merge then?